Dalamud/Game/ClientState/JobGauge/Enums/SerpentCombo.cs

```diff
@@ -0,0 +1,42 @@
+namespace Dalamud.Game.ClientState.JobGauge.Enums;
+
+/// <summary>
+/// Enum representing the SerpentCombo actions for the VPR job gauge.
+/// </summary>
+public enum SerpentCombo : byte
```

Comment:
Please use `PascalCase` for enum members.

Reply:
I'm following the convention of many other enums in this folder, even as recently as `BeastChakra.cs`. Only the PCT and other VPR enums use `PascalCase`.

Reply:
I would still prefer we clean up and move everything over to PascalCase moving forward, I'm guessing we missed a fair bit.

That said, I'll merge this change in as-is, and we can schedule a cleanup in a future API version.